spec/controllers/api/v1/preferences_controller_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::PreferencesController, type: :controller do
+  let!(:user1) { create(:github_user, login: "user_login") }
+  let!(:user2) { create(:github_user, login: "user_login2") }
+  let(:github_session) { instance_double("GithubSession") }
+  let(:preference1) { create(:preference, github_user: user1) }
+  let(:preference2) { create(:preference, github_user: user2) }
+
+  before do
+    allow(github_session).to receive(:token).and_return("token")
+    allow(github_session).to receive(:user).and_return(user1)
+    allow(subject).to receive(:github_session).and_return(github_session)
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>RSpec/SubjectStub: Do not stub methods of the object under test.

Reply:
https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/SubjectStub